### PR TITLE
Example for Elastic Beanstalk

### PR DESCRIPTION
**Pull request by fss18**
added example for Elastic Beanstalk multiple containers.
this example will include:
1. container definition
2. ebextension example to collect syslog on ec2 level

### CODE REVIEW

File: elasticbeanstalk/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**To find your unique registration key:**
+
+1. In the Alert Logic console, click the Support Information icon.
+2. Click "Details."
+3. Copy your unique registration key.
+
+## Alert Logic Agent Container Definition
+To deploy the Agent Container for EB, you must:
+1. Include the example container definition file from this repository and merge it to your own EB container configuration file (`Dockerrun.aws.json`). The Agent Container will run as side-car container along with your application container.
+2. Include the example `.ebExtensions` configuration and merge it to your own EB application source bundle.
```
Comment (ericreeves) [inline elasticbeanstalk/README.md]: Capital E, or lowercase?

File: README.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
-# Alert Logic Agent Container for Kubernetes, ECS, and Docker
+# Alert Logic Agent Container for Kubernetes, ECS, EB, and Docker
```
Comment (ericreeves) [inline README.md]: Small detail, but shall we spell out "Elastic Beanstalk" here since it's the title?

Comment (fss18) [reply README.md]: agree, EB is not popular enough!

**Comment by fss18** [comment]
@eric-al , see https://github.com/alertlogic/al-agent-container/pull/22/commits/f859da38391adea7e8a253ac0fab81b58abe328e for the typo and naming convention.

**Review by ericreeves — APPROVED** [review]

**Comment by ericreeves** [comment]
This looks reasonable to me.  Let's maybe have @jnorden take a peek, and have @rhosmo or someone on her team take a look at it before we merge.

File: README.md (hunk 2 above)
Comment (rhosmo) [reply README.md]: According to AWS, it's "AWS Elastic Beanstalk" on the first reference, and "Elastic Beanstalk" going forward from there. I can perform an edit for grammar and for AL Info Dev style standards.

**Comment by jnorden** [comment]
I say once @rhosmo does her review and gives it a "thumbs-up" we're good.  I have nothing to add

**Comment by fss18** [comment]
@jnorden I got thumbs up from @rhosmo , 
she may do an extra grammar check on a separate PR

**Pull request merged**